content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,149 @@
+
+Because the participants want to disallow the escrow from being finished until after 11:00 pm the night of the party (the time when the party band is scheduled to finish its set), the party host must calculate a `FinishAfter` value to include in the escrow definition.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Create the escrow](send-a-conditionally-held-escrow.html#3-submit-escrowcreate-transaction) (sender)
+
+The party host (sender) creates the escrow that provides the smart contract. The party host must create the escrow because they are the only participant that can authorize the lock up and potential payout of XRP from their XRP Ledger account.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Wait for validation](send-a-conditionally-held-escrow.html#4-wait-for-validation) and [confirm escrow creation](send-a-conditionally-held-escrow.html#5-confirm-that-the-escrow-was-created) (sender and receiver)
+
+The party host (sender) waits for validation of the ledger that contains the escrow creation transaction and then confirms that the escrow was created.
+
+The party host then provides the escrow transaction's `hash` value to the party band (receiver). The party band can use the `hash` value to look up the escrow transaction on the XRP Ledger to ensure that it was created according to the smart contract terms they agreed to.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Finish the escrow](send-a-conditionally-held-escrow.html#6-submit-escrowfinish-transaction) (oracle)
+
+The party band (receiver) shows up and plays their set.
+
+The party planner (oracle) finishes the escrow using the fulfillment value they generated and kept secret.
+
+Alternatively, if the party band is a no-show, the party planner does not finish the escrow. In this case, after 12 noon the next day, any participant can [cancel the escrow](cancel-an-expired-escrow.html) to return the held XRP to the party host's (sender's) account.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Wait for validation](send-a-conditionally-held-escrow.html#7-wait-for-validation) and [confirm escrow finish](send-a-conditionally-held-escrow.html#8-confirm-final-result) (oracle, sender, and receiver)
+
+The party planner (oracle) waits for validation of the ledger that contains the escrow finish transaction and then confirms that the escrow was finished.
+
+At this time, the party planner provides the transaction's `hash` value to the party host (sender) and party band (receiver). The party host and party band can use the `hash` value to look up the escrow transaction on the XRP Ledger to ensure that it is been finished correctly. The party host and party band can also just check their XRP Ledger account balances to ensure that their balances have decreased and increased by 2000 XRP respectively.
```

Comment:
The host's balance won't change at this step (unless the escrow was canceled) because the escrow creation already debited the locked-up XRP from their account.